### PR TITLE
Added QMTECH RP2040 Daughterboard

### PR DESCRIPTION
**Pull request by Chandler-Kluser**
Added new QMTECH Daughterboard with RP2040, which can be found in [AliExpress](https://www.aliexpress.com/item/1005005094654777.html).

It has some differences compared to the previous model.

Documentation can be found [here](https://github.com/ChinaQMTECH/DB_FPGA_with_RP2040).

### CODE REVIEW

File: litex_boards/targets/qmtech_rp2040_daughterboard.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+            11: "J2:52",
+            12: "J2:51",
+            13: "J2:50",
+            14: "J2:49",
+            15: "J2:48",
+            16: "J2:47",
+            17: "J2:46",
+            18: "J2:45"
+        }),
+    ]
```
Comment (josuah) [inline litex_boards/targets/qmtech_rp2040_daughterboard.py:136]: Often, the list of I/Os and connectors go to a separate "platform/...py" source file.

Is it intended to keep it here?

Comment (Chandler-Kluser) [reply litex_boards/targets/qmtech_rp2040_daughterboard.py:136]: I did not know, I will drop another commit

Comment (josuah) [reply litex_boards/targets/qmtech_rp2040_daughterboard.py:136]: Maybe moving `litex_boards/targets/qmtech_rp2040_daughterboard.py` to `litex_boards/platforms/qmtech_rp2040_daughterboard.py` is enough? As there is not really possible to run this script and get a board out of it.

Comment (Chandler-Kluser) [reply litex_boards/targets/qmtech_rp2040_daughterboard.py:136]: yes, I am doing that!

**Review by Chandler-Kluser — COMMENTED** [review]
Fixed Issue

**Comment by enjoy-digital** [comment]
Thanks @Chandler-Kluser, that's merged.

**Pull request merged**